test/runtests.jl

```diff
@@ -44,7 +46,7 @@ include("helpers.jl")
     include("utils.jl")
     include("class.jl")
     include(joinpath("tzdata", "timeoffset.jl"))
-    include(joinpath("tzdata", "archive.jl"))
+    VERSION < v"1.4" && include(joinpath("tzdata", "archive.jl"))
```

Comment:
I realize this isn't needed on 1.4 but is there a reason no to include it?

Reply:
The `archive.jl` is testing the contents of `ARCHIVE_DIR` and whether it stored archives. When I run it on 1.4, which is not storing archives here, because they are stored in already unpacked form (Pkg.Artifacts unpacks the archivs automatically) in artifacts directory, this tests is failing, see: https://travis-ci.com/github/JuliaTime/TimeZones.jl/jobs/361057661

Reply:
Ok, I get why you're disabling these tests now.